src/main.ts

```diff
@@ -1,10 +1,46 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import {
+  SwaggerModule,
+  DocumentBuilder,
+  SwaggerCustomOptions,
+} from '@nestjs/swagger';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+
+  // configure swagger
+  const swaggerConfig = new DocumentBuilder()
+    .setTitle('Edu Server')
+    .setDescription('Edu Server API documentation')
+    .setVersion('0.0.1')
+    .setContact(
+      'Code for Cause',
+      'https://codeforcause.org/',
+      'team@codeforcause.org',
+    )
+    .setLicense(
+      'MIT',
+      'https://github.com/codeforcauseorg/edu-server/blob/master/LICENSE',
+    )
+    .addServer('http://localhost:5000/', 'Development Server')
+    .addBearerAuth()
+    .build();
+
+  // adding custom options
+  const customOptions: SwaggerCustomOptions = {
+    swaggerOptions: {
+      persistAuthorization: true,
+    },
+    customCss: '.swagger-ui .topbar { display: none }',
+    customSiteTitle: 'Edu Server API Docs',
+    customfavIcon: 'https://codeforcause.org/favicon.ico',
+  };
```

Comment:
Add custom options in that file too, and export these constants

Reply:
should i create a docs folder in the root of the project and add the swagger file in that folder? @Abhishek-kumar09

Reply:
create a swagger folder in src and create a new file to put these.